nwb_conversion_tools/gui/nwb_conversion_gui.py

```diff
@@ -25,12 +25,14 @@
 import yaml
 import sys
 import os
-
+import inspect
+# TODO: implement changes specified downstread
+#   -check kwargs
 
 class Application(QMainWindow):
     def __init__(self, metafile=None, conversion_module='', source_paths={},
                  kwargs_fields={}, extension_modules={}, extension_forms={},
```

Comment:
Null default values should be `None`. This applies to `conversion_module` and all of the subsequent `dict`s. I'd also like to mark conversion_module as depricated.